core/src/software/aws/toolkits/core/telemetry/Metric.kt

```diff
@@ -0,0 +1,84 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.core.telemetry
+
+import software.aws.toolkits.core.utils.getLogger
+import java.time.Duration
+import java.time.Instant
+import java.time.ZoneOffset
+import java.time.ZonedDateTime
+import java.util.Collections
+import java.util.concurrent.ConcurrentHashMap
+import java.util.concurrent.atomic.AtomicBoolean
+
+/**
+ * Event that occurred in the Toolkit.
+ */
+open class Metric internal constructor(internal val metricNamespace: String, private val publisher: MetricsPublisher) :
+    AutoCloseable {
+    private val closed = AtomicBoolean(false)
+    private val _entries: MutableMap<String, MetricEntry> = ConcurrentHashMap()
```

Comment:
Style: remove the underscore since it is already a private member

Reply:
✔️